man/unison.1.in

```diff
@@ -310,9 +310,23 @@ or
 .Sy SIGTERM
 works the same way as described above and will additionally stop the continuous
 process. To stop only the continuous process and let the last synchronization
-complete normally, send signal
+complete normally, press
+.Sy Ctrl-D
+or send signal
 .Sy SIGUSR2
 instead.
+Closing the input, receiving an EOF, or receiving a
```

Comment:
I really do not understand "closing the input".   stdin is file descriptor zero and unison is in charge of closing/not-closing that.  I think you mean closing the other half of some split pty/fifo special file to cause read(2) in unison to return EOF.

Reply:
I think you're right. There may have been some situation where a close on the other end would not produce EOF on the read end but I can't remember what it was. And thinking about it now, that doesn't seem realistic either. I may have been trying to make the language assume less about the reader knowing about conditions when EOF is generated. I'll see if I can rephrase it (no promises).

Reply:
I would prefer to speak precisely about how unison behaves based on return values from read(2) syscall on fd 0, and not try to have it make sense to people that do not understand, when that has any fuzziness.